module/entities/helper.js

```diff
@@ -651,6 +661,7 @@ module.exports = class EntitiesHelper {
     static processEntityMappingUploadData(mappingData = []) {
         return new Promise(async (resolve, reject) => {
             try {
+                let childEntityId = [];
```

Comment:
@deepakr123 - Rename it to entities please

Reply:
resolved